src/App.vue

```diff
@@ -490,6 +492,13 @@ export default {
       this.setLastAddressChangeDate(business.lastAddressChangeDate) // may be empty
       this.setLastDirectorChangeDate(business.lastDirectorChangeDate) // may be empty
 
+      // *** TODO: get entity status (eg, Dissolved) -- see also business.goodStanding
+
+      // *** TODO: remove debugging code
+      business.dissolutionDate = this.dateToApi(new Date())
+      this.setEntityDissolutionDate(this.apiToDate(business.dissolutionDate))
+      this.setEntityDissolutionType(DissolutionTypes.HDO) // *** TODO: get from business object
```

Comment:
I'm very tempted to merge in this code as-is. The functionality is disabled in the `isHistorical` getter anyway (which needs to be wired up however we get data from the API). This is so my code based doesn't fall behind. Thoughts?

Reply:
@argush3 Any thoughts on this? Linda was OK with it at s/u this morning. I'm just waiting for Cam's and your final reviews.

Reply:
yup, i'm good with merging the code into main so it doesn't go stale.  will try to get my review in before EOD.